#build/Publish-All.ps1

```diff
@@ -79,6 +79,15 @@ function Get-PredefinedBranches() {
     })
 }
 
+function Set-Content-UTF8($file, $content) {
+    if ($HOST.Version.Major -ge 6) {
+        # PowerShell Core 6.0+ natively support UTF-8 output but no more -Encoding Byte support
+        return Set-Content $file $content
+    } else {
+        return Set-Content $file -Encoding Byte ([Text.Encoding]::UTF8.GetBytes($content))

```

Comment:
Can this use Set-Content -Encoding UTF8 (and is that still supported in PowerShell 6)?

Reply:
`Set-Content -Encoding UTF8` is still supported in PowerShell 6, and it's equivalent to `Set-Content -Encoding` which write out UTF8N string.
But `Set-Content -Encoding UTF8` in PowerShell -5 generates UTF8 (w/ BOM), which I assume is not suitable for modern Web. My assumption might be incorrect.